dist/Exporter/t/Exporter.t

```diff
@@ -26,7 +26,7 @@ BEGIN {
 
 BEGIN {
     # Methods which Exporter says it implements.
-    @Exporter_Methods = qw(import
+    our @Exporter_Methods = qw(import
```

Comment:
I think you want to define `our @Exporter_Methods;` before the `BEGIN` block
then set it inside the `BEGIN block

Reply:
Done.